python/pyarrow/_fs.pyx

```diff
@@ -166,19 +208,36 @@ cdef class FileInfo(_Weakrefable):
         Returns
         -------
         type : FileType
+
+        Examples
+        --------
+        >>> file_info = local.get_file_info(path)
+        >>> file_info.type
+        <FileType.File: 2>
```

Comment:
I don't know, do you think this example is useful? The method is trivial enough that I don't think this is showcasing something interesting. @jorisvandenbossche What do you think?

Reply:
I can remove them without problem ... they are mostly duplicated in the class docstring anyways. That holds for most of the comments ⬇️ 🤣